ui/src/components/credentials/LinksTable.tsx

```diff
@@ -0,0 +1,401 @@
+          disabled={link.status === "exceeded"}
+          loading={isLinkUpdating[link.id]}
+          onClick={(isActive) => {
+            toggleLinkActive(isActive, link.id);
+          }}
+          size="small"
+        />
+      ),
+      sorter: ({ active: a }, { active: b }) => (a === b ? 0 : a ? 1 : -1),
+      title: "Active",
+      width: 100,
+    },
+    {
+      dataIndex: "schemaType",
+      ellipsis: true,
+      key: "schemaType",
+      render: (schemaType: Link["schemaType"]) => (
+        <Tooltip placement="topLeft" title={schemaType}>
+          <Typography.Text strong>{schemaType}</Typography.Text>
+        </Tooltip>
+      ),
+      sorter: ({ schemaType: a }, { schemaType: b }) => a.localeCompare(b),
+      title: "Credential",
+    },
+    {
+      dataIndex: "expiration",
+      ellipsis: true,
+      key: "expiration",
+      render: (expiration: Link["expiration"]) => (
+        <Typography.Text>{expiration ? formatDate(expiration, true) : "Unlimited"}</Typography.Text>
+      ),
+      sorter: ({ expiration: a }, { expiration: b }) => {
+        if (a && b) {
+          return dayjs(a).unix() - dayjs(b).unix();
+        } else if (a) {
+          return 1;
+        } else {
+          return -1;
+        }
+      },
+      title: ACCESSIBLE_UNTIL,
+    },
+    {
+      dataIndex: "issuedClaims",
+      ellipsis: true,
+      key: "issuedClaims",
+      render: (issuedClaims: Link["issuedClaims"]) => {
+        const value = issuedClaims ? issuedClaims : 0;
+
+        return <Typography.Text>{value}</Typography.Text>;
+      },
+      sorter: ({ issuedClaims: a }, { issuedClaims: b }) => (a === b ? 0 : a ? 1 : -1),
+      title: "Credentials issued",
+    },
+    {
+      dataIndex: "maxIssuance",
+      ellipsis: true,
+      key: "maxIssuance",
+      render: (maxIssuance: Link["maxIssuance"]) => {
+        const value = maxIssuance ? maxIssuance : "Unlimited";
+
+        return <Typography.Text>{value}</Typography.Text>;
+      },
+      sorter: ({ maxIssuance: a }, { maxIssuance: b }) => {
+        if (a && b) {
+          return a - b;
+        } else if (a) {
+          return 1;
+        } else {
+          return -1;
+        }
+      },
+      title: "Maximum issuance",
+    },
+    {
+      dataIndex: "status",
+      key: "status",
+      render: (status: Link["status"]) => (
+        <Row justify="space-between">
+          {(() => {
+            switch (status) {
+              case "active": {
+                return <Tag color="success">Active</Tag>;
+              }
+              case "inactive": {
+                return <Tag>Inactive</Tag>;
+              }
+              case "exceeded": {
+                return <Tag color="error">Exceeded</Tag>;
+              }
+            }
+          })()}
+          <Dropdown
+            menu={{
+              items: [
+                {
+                  icon: <IconInfoCircle />,
+                  key: "details",
+                  label: "Details",
```

Comment:
Are we missing an issue for this?

Reply:
Created issues for [details](https://polygonid.atlassian.net/browse/PID-562?atlOrigin=eyJpIjoiNjUwNGFjMWE3Y2I2NDQ5OThmODk0N2JkNmExYTg0MWEiLCJwIjoiaiJ9) & [deletion](https://polygonid.atlassian.net/browse/PID-563?atlOrigin=eyJpIjoiY2UyNzE4M2YwZDlhNDIzMWE4MzUzNDQ5M2NjYWQ2MzYiLCJwIjoiaiJ9).